.github/workflows/check-go-versions.yml

```diff
@@ -0,0 +1,85 @@
+name: Check Supported Go Versions
+on:
+  schedule:
+    - cron: "0 17 * * *"
+  workflow_dispatch:
+
+jobs:
+  check-go-eol:
```

Comment:
We can do this later. But now that this is used in multiple repos, could put it in gh-actions.